scripts/Samuel-Ajayi.js

```diff
@@ -1,4 +1,4 @@
 function	hng(name, id,language, email) {
-	console.log(`Hello world, this is ${name} with HNGi7 ID ${id} and ${email} using ${language} for stage 2 task`)
+	console.log(`Hello world, this is ${hng.name} with HNGi7 ID ${id} and ${email} using ${language} for stage 2 task`)
```

Comment:
console.log(`Hello World, this is ${hng.name} with HNGi7 ID ${id} and email ${email} using ${language} for stage 2 task`)
Yours should look like this. update it fast.

Reply:
thanks